scripts/logging.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from modules import shared
+
+# Create a new logger
+logger = logging.getLogger("ControlNet")
+
+# Configure logger
+handler = logging.StreamHandler()
+handler.setFormatter(
+    logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+)
+logger.addHandler(handler)
+
+
+if hasattr(shared.cmd_opts, "controlnet_loglevel"):
+    loglevel_string = shared.cmd_opts.controlnet_loglevel
+else:
+    print("Warning: `controlnet_loglevel` not available from A1111")
+    loglevel_string = "DEBUG"
```

Comment:
Why not only use `getattr(shared.cmd_opts, "controlnet_loglevel", "INFO")`? AFAIK the if branch should only run when users do not do a full restart of the webui. Also, is there a reason we are falling back on debug here but using info by default?

Reply:
Done.